frontend/app/src/components/Sidebar/Sidebar.tsx

```diff
@@ -221,9 +224,12 @@ class Sidebar extends PureComponent<SidebarProps, State> {
             isCollapsed={collapsedSidebar}
             data-testid="collapsedControl"
           >
-            <Button kind={Kind.HEADER_BUTTON} onClick={this.toggleCollapse}>
+            <BaseButton
+              kind={BaseButtonKind.HEADER_BUTTON}
+              onClick={this.toggleCollapse}
+            >
               <Icon content={ChevronRight} size="lg" />
-            </Button>
+            </BaseButton>
```

Comment:
Renamed because there was a conflict with Button Widget that is being exported as of now.